### ISSUE TITLE
Make iostreams component work locally

### ISSUE THREAD

**Issue by msimberg** [STEllAR-GROUP/hpx#4565]
Either conditionally make it work with the local runtime, or add another implementation that always works locally.

**Comment by toktarev** [comment]
Is this issue still actual ?

**Comment by hkaiser** [comment]
> Is this issue still actual ?

Yes! The question would be whether a simple mapping `hpx::cout` --> `std::cout` would already do the trick. I.e. something like a simple conditional
```
namespace hpx {
    using cout = std::cout;
}
```
